playbooks/roles/filebeat/defaults/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+#logstash_ssl_dir: /etc/pki/logstash
+#logstash_forwarder_ssl_certificate_file: logstash-forwarder-example.crt
+
+filebeat_app_dir: "{{ COMMON_APP_DIR }}/filebeat"
+filebeat_cfg_dir: "{{ COMMON_CFG_DIR }}/filebeat"
+filebeat_file: "filebeat_1.2.3_amd64.deb"
```

Comment:
As with the other roles, let's split out the version into it's own variable,
so that it's easier to update later.

```yaml
LOGSTASH_FILEBEAT_VERSION: '1.2.3'
logstash_filebeat_file: "filebeat_{{ LOGSTASH_FILEBEAT_VERSION }}_amd64.deb"
```

The convention in this repo is that all-caps variable names are intended to be user-overridden at deploy time, when needed.